src/Components/OATModelList/OATModelList.tsx

```diff
@@ -116,6 +116,16 @@ const OATModelList = ({
         setFilter(evt.target.value);
     };
 
+    const getDisplayNameText = (item) => {
+        return typeof item['displayName'] === 'string'
```

Comment:
Rather than indexing into item to get display name each time, maybe pull it out first?

E.g. `const displayName = item['displayName'];`.